### PR TITLE
[PERF-317] Run Paging Test on github Action

### PR DESCRIPTION
**Pull request by JBrenesSimpat**

### CODE REVIEW

File: .github/workflows/Package_pre_release.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-name: Package Pre-Release-Code
+name: Package Pre-Release
```
Comment (semalaiappan) [inline .github/workflows/Package_pre_release.yml:6]: Avoid Underscore in file name for any file under workflows

File: .github/workflows/Paging_test.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Run Paging Test
+on:
+  workflow_dispatch:
+
+env:
+    AzureTestVmUsername: ${{ secrets.AZURE_TEST_VM_USER_NAME }}
```
Comment (semalaiappan) [inline .github/workflows/Paging_test.yml:11]: Use underscore here . example AZURE_TEST_VM_USERNAME: ${{ secrets.AZURE_TEST_VM_USER_NAME }}

File: .github/workflows/Paging_test.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608  # v4.1.0
+
+      - name: Execute Test Runner
+        if: ${{ env.AzureTestVmUsername != '' && env.AzureTestVmPassword != '' }}
+        shell: powershell
+        working-directory: ./eng/
+        run: |
+
```
Comment (semalaiappan) [inline .github/workflows/Paging_test.yml:27]: Remove whitespaces lines here

**Review by semalaiappan — CHANGES_REQUESTED** [review]
Added my review comments

**Review by vimayya — APPROVED** [review]

File: .github/workflows/run_paging_test.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        if: ${{ env.AzureTestVmUsername != '' && env.AzureTestVmPassword != '' }}
+        shell: powershell
+        working-directory: ./eng/
+        run: |
+            . $env:GITHUB_WORKSPACE/eng/AzureTestLab.ps1
+            Invoke-TestRunnerFromTeamCity pageVolume
+
+      - name: Upload Package
+        if: success()
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
```
Comment (semalaiappan) [inline .github/workflows/run_paging_test.yml:34]: Can you update to actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.3.0 this one also ?

**Review by semalaiappan — CHANGES_REQUESTED** [review]

**Review by semalaiappan — APPROVED** [review]

**Pull request merged**